README.rst

```diff
@@ -1,29 +1,14 @@
-Astropy package template
-========================
+Astropy Package Template
+------------------------
 
 .. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
-This is a package template provided by the Astropy project.
+This is a rendered version of the cookiecutter Astropy package template, which you can find `here <https://github.com/astropy/package-template>`_. You can also read the `documentation <http://docs.astropy.org/projects/package-template>`_ for more details.
 
-Using this template, packages can make use of the setup, installation, and documentation
-infrastructure developed for the ``astropy`` core and affiliated packages.
-
-For more information, see:
-
-* `Detailed instructions for using this template <http://docs.astropy.org/en/latest/development/astropy-package-template.html>`_
-* `The Affiliated Packages section of the Astropy web site <http://affiliated.astropy.org>`_
-* `This template's Github code repository <https://github.com/astropy/package-template>`_
-
-Status reports for developers
------------------------------
-
-.. image:: https://travis-ci.org/astropy/package-template.svg
-    :target: https://travis-ci.org/astropy/package-template
-    :alt: Travis Status
 
 License
 -------
 
-This project is Copyright (c) Astropy Developers and licensed under the terms of the BSD 3-Clause license. See the licenses folder for more information.
+This project is Copyright (c) Astropy Developers and licensed under the terms of the Other license. See the licenses folder for more information.
```

Comment:
@Cadair - the 'Other' licence - huh? :)

Reply:
Well the master branch dosen't have a licence chosen, this for clarity is not the licence of the template.

(Cadair goes and add the template file)

Reply:
see #322

Reply:
I still think this sentence should mabye say 'the license in LICENSE.rst' or something, 'Other' sounds weird

Reply:
I am not sure that will be very easy given the way we template this out. Can we punt this to an issue?

Reply:
Yep!

Reply:
But please open the issue :)

Reply:
I agree with @astrofrog - we can't punt this because technically people can't use this if we don't get this right.  But aren't we OK if #322 is merged to do what @astrofrog suggests?  (Although it might make sense to rename it to `TEMPLATE_LICENSE` or something like that to avoid the ambiguity?)

Reply:
The idea of this sentence is the licence for the *package* not for the template. The template licence is well defined since #322 I will add an extra sentence to the README about the template licence.